frontend/pages/hosts/HostDetailsPage/_styles.scss

```diff
@@ -1,17 +1,24 @@
 .host-details {
   display: flex;
-  flex-direction: column;
+  flex-wrap: wrap;
```

Comment:
```suggestion
  flex-wrap: wrap;
  flex-grow: 1;
  align-content: flex-start;
```
The `ui-off-white` doesn't always fill up the entire page's height. See the screenshot below for an example of this behavior (I'm having to squint to see the difference lol):
![localhost_8080_hosts_1](https://user-images.githubusercontent.com/47070608/116927908-52a3c300-ac2a-11eb-8d51-8895a245bfbe.png)

A suggested solution is to add the `flex-grow` and `align-content` properties but there could definitely be a better solution. Flexbox is awesome.

Reply:
Thanks for all the input-- should be good to go soon!